Makefile

```diff
@@ -22,14 +22,15 @@ $(PKGS): golint
 	@gofmt -w=true $(GOPATH)/src/$@*/**.go
 ifneq ($(NOLINT),1)
 	@echo "LINTING..."
-	@PATH=$(PATH):$(GOPATH)/bin golint $(GOPATH)/src/$@*/**.go
+	@PATH=$(PATH):$(GOPATH)/bin golint $(GOPATH)/src/$@/...
```

Comment:
this will lint all sub-packages as well, which could lead to some redundant output

I'd follow the pattern we have in other Go Makefiles and enumerate the packages we're testing, i.e.

```
SUBPKGS = $(addprefix $(PKG)/, cmd/csvlint)
PKGS = $(PKG) $(SUBPKGS)
```

then the original form of the command here should work